Jenkinsfile

```diff
@@ -130,7 +130,8 @@ def buildUnitTestStage(env) {
             sh 'envsubst < ci/helm/nuxeo-test-base-values.yaml > nuxeo-test-base-values.yaml'
             def testValues = '--set-file=nuxeo-test-base-values.yaml'
             if (!isDev) {
-              testValues += " --set-file=ci/helm/nuxeo-test-${env}-values.yaml"
+              sh "envsubst < ci/helm/nuxeo-test-${env}-values.yaml > nuxeo-test-${env}-values.yaml"
```

Comment:
What if the file does not exist? Dunno if we will have the case...

Reply:
Shouldn't have the case, we assume we're maintaining the values files according to the env list/